docs/how-to-setup-freecodecamp-locally.md

```diff
@@ -255,13 +255,16 @@ npm run develop
 This single command will fire up all the services, including the API server and the client applications available for you to work on.
 
 > [!NOTE]
-> Once ready, open a web browser and **visit <http://localhost:8000>**. If the app loads, congratulations – you're all set! You now have a copy of freeCodeCamp's entire learning platform running on your local machine.
+> Once ready, open a web browser and **visit <http://localhost:8000>**. If the app loads, sign in. Congratulations – you're all set! You now have a copy of freeCodeCamp's entire learning platform running on your local machine.
 
 > [!TIP]
 > The API Server serves APIs at `http://localhost:3000`. The Gatsby app serves the client application at `http://localhost:8000`
 
 > If you visit <http://localhost:3000/explorer> you should see the available APIs.
```

Comment:
```suggestion
> Once ready, open a web browser and **visit <http://localhost:8000>**. If the app loads, congratulations – you're all set! You now have a copy of freeCodeCamp's entire learning platform running on your local machine.

> [!TIP]
> The API Server serves APIs at `http://localhost:3000`. The Gatsby app serves the client application at `http://localhost:8000`

> If you visit <http://localhost:3000/explorer> (AFTER signing in) you should see the available APIs.
```
I wonder if it would be better to add the note here?

Reply:
If I didn't misunderstanding the tip, you need to log in `localhost:8000`. my issue-as not native english speaker-with adding it to line 263, it maybe misunderstood that you need to log into `localhost:3000`.

but I think I should add

```md
While you are logged in, If you visit <http://localhost:3000/explorer> (AFTER signing in) you should see the available APIs.
```

what do you think about this?

Reply:
Yes, though we'd remove the `(after signing in)` text. :)